test/e2e/utils.go

```diff
@@ -548,3 +550,37 @@ func deletePersistentVolumeClaims(namespace string) {
 		fw.KubeClient.CoreV1().PersistentVolumeClaims(kafkaNamespace).Delete(pvc.Name, &emptyDeleteOptions)
 	}
 }
+
+func verifyCollectorImage(jaegerInstanceName, namespace string, expected bool) {
+	if expected {
```

Comment:
The if statement can be omitted there is `require.Equal(t, expected, got)`.  Then this function becomes a one-liner :)